dynamiq/nodes/tools/zenrows.py

```diff
@@ -9,33 +9,34 @@
 from dynamiq.utils.logger import logger
 
 
+class ZenRowsInputSchema(BaseModel):
+    url: str = Field(..., description="Parameter to provide a url of the page to scrape.")
+
+
 class ZenRowsTool(ConnectionNode):
     """
     A tool for scraping web pages, powered by ZenRows.
 
     This class is responsible for scraping the content of a web page using ZenRows.
-    The input should be a dictionary containing the key 'input', which includes the URL of the page to scrape.
     """
 
     group: Literal[NodeGroup.TOOLS] = NodeGroup.TOOLS
-    name: str = "Scraper-ZenRows"
+    name: str = "Zenrows Scraper Tool"
     description: str = (
         "A tool for scraping web pages, powered by ZenRows. "
         "You can use this tool to scrape the content of a web page."
-        "Input should be a dictionary with a key 'input' containing the URL of the page to scrape."
     )
     connection: ZenRows
-    url: str | None = None
```

Comment:
Please revert this one. It usable by lib and also exists on UI. Also please fix `ZenRowsInputSchema.url` to make it optional